packages/core/src/components/tooltip/tooltip.tsx

```diff
@@ -114,13 +115,36 @@ export class Tooltip<
         this.popoverRef.current?.reposition();
     }
 
+    protected validateProps(props: TooltipProps<T>) {
+        const childrenCount = React.Children.count(props.children);
+        if (childrenCount > 1) {
+            console.warn(Errors.POPOVER_WARN_TOO_MANY_CHILDREN);
+        }
+        // all other warnings should occur in Popover, not here.
+    }
+
     // any descendant ContextMenus may update this ctxState
     private renderPopover = (ctxState: TooltipContextState) => {
-        const { children, compact, disabled, intent, popoverClassName, ...restProps } = this.props;
+        const { children, content, renderTarget, compact, disabled, intent, popoverClassName, ...restProps } =
+            this.props;
+
         const popoverClasses = classNames(Classes.TOOLTIP, Classes.intentClass(intent), popoverClassName, {
             [Classes.COMPACT]: compact,
         });
 
+        const contentElement = Utils.ensureElement(content);
+        const tooltipId = contentElement?.props?.id ?? Utils.uniqueId("tooltip");
+        const clonedContent = contentElement
+            ? React.cloneElement(contentElement, { role: "tooltip", id: tooltipId })
+            : contentElement;
+
+        const childTarget = Utils.ensureElement(React.Children.toArray(children)[0]);
+        const clonedTarget = childTarget
```

Comment:
if `childTarget` is defined with `ensureElement` then I think this next line will always be truthy which I don't think we want

Reply:
Nope, it will be falsy. `Utils.ensureElement`:
![image](https://github.com/palantir/blueprint/assets/87083504/8a199bdb-948a-459b-b31c-0e4ac853b4e9)